sap-s4hana-cloud/2.2/modules/ROOT/pages/sap-s4hana-cloud-connector-studio.adoc

```diff
@@ -0,0 +1,151 @@
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Anypoint Studio, create a new Mule project in which to add and configure the connector:
+
+. From the Studio main menu, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Anypoint Connector for SAP S/4HANA OData to a Mule project to automatically populate the project's XML code with the connector's namespace and schema location, as well as to add the required dependencies to the project's `pom.xml` file.
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Dependencies to Project*, type `SAP S/4HANA OData` in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met. The act of dragging the connector's operation from the palette onto the Anypoint Studio canvas automatically populates the XML code with the connector namespace and schema location.
+
+Possible input source operations for this connector are:
+
+* HTTP Listener +
+Starts your app using a browser or a command such as `curl`
+* Scheduler +
+Starts your app at timed intervals
+
+In Studio, drag the input operation you choose from the *Mule Palette* to the *Source* panel in the canvas, and supply the required parameters.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for SAP S/4HANA OData Connector, follow these steps:
+
+. In the *Mule Palette* view, select *SAP S/4HANA OData* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Global Configuration Elements
+
+To use SAP S/4HANA OData Connector in a Mule app, configure a connector global element for connector operations to use in your application.
+
+You can put placeholder values in the `mule-artifact.content` configuration file in the `src/main/resources` folder of your project.
+
+You can either enter your credentials in the global configuration content or
+reference a configuration file that contains the values for your credentials.
+Using a configuration file is recommended for simpler maintenance and better reuse of your project. Keeping these values in a separate file is useful if you need to deploy to different environments, such as production, development, and QA, where access credentials might differ.
```

Comment:
This may not have made it into the template yet, but I have been adding "For more information..." and  linking to the Use Property Placeholders for Property Values section of the Anypoint Connector Configuration topic because it has instructions for referencing the configuration file.